src/stream-chat-net/IChannel.cs

```diff
@@ -28,7 +28,7 @@ public interface IChannel
         Task<TruncateResponse> Truncate();
         Task<TruncateResponse> Truncate(TruncateOptions truncateOptions);
 
-        Task AddMembers(IEnumerable<string> userIDs, MessageInput msg = null);
+        Task AddMembers(IEnumerable<string> userIDs, MessageInput msg = null, bool hideHistory = false);
```

Comment:
this looks fine but it would be better to introduce an options object to make it extensible

Reply:
agreed, its a breaking change though

Reply:
maybe limit it to last param and keep others same